README.md

```diff
@@ -1,135 +1,61 @@
+- You will need to download and run an Elastisearch server:
 
-Event emitted publically are published via a broker topic named `item-ItemEvent`.
+```
+curl -L -O https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-5.0.2.tar.gz
+tar -xvf elasticsearch-5.0.2.tar.gz
+cd elasticsearch-5.0.2/bin
+./elasticsearch
+```
+### Running
 
-## Commands handled
+On another terminal, clone this repo and run the sample app using the command `sbt runAll`:
 
-### External (user)
+```
+git clone git@github.com:lagom/online-auction-scala.git
+cd online-auction-scala
+sbt runAll
+```
 
-* **createItem** - Creates an item - emits **ItemUpdated**.
-* **updateItem** - Updates user editable properties of an item, if allowed in the current state (eg, currency can't be updated after auction is started), emits **ItemUpdated**.
-* **startAuction** - Starts the auction if current state allows it, emits **AuctionStarted**.
-* **cancelAuction** (not supported) - Cancels the auction if current state allows it, emits **AuctionCancelled**.
+# Auction system - System architecture
 
-## Events consumed
+The auction system is the sum of 5 micro-services and a web gateway:
 
-### Bidding service
+* **[Item Service](docs/item-service.md)**: Manages the description and auction status (created, auction, completed, cancelled) of an item.
+* **[Bidding service](docs/bidding-service.md)**: Manages bids on items.
+* **[Search service](docs/search-service.md)**: Handles all item searching.
+* **[Transaction service](docs/transaction-service.md)**: Handles the transaction of negotiating delivery info and making payment of an item that has completed an auction.
+* **[User service](docs/user-service.md)**: a convenience service to stub user management. Don't use any code in `user-service` as reference on how to create a secure user management micro-service.
+* **web-gateway**: a [Play](https://www.playframework.com/) application providing web UI and acting as gateway to all previously described services.
 
-* **BidPlaced** - Updates the current price of the item.
-* **BiddingFinished** - Completes the auction if current state allows, emits **AuctionFinished**.
+Check the docs for each service for details on the **commands** and **queries** it serves as well as events the service **emits** and events it **consumes** from the [Message Broker](http://www.lagomframework.com/documentation/1.3.x/scala/MessageBrokerApi.html#Message-Broker-Support).
 
-# Bidding service
+## Good to know
 
-Manages bids on items.
+- System requirements
 
-## Queries handled
+We recommend using a decent machine and having at least 8GB of RAM for development.
 
-* **getBids** - Gets all the bids for an item.
+- Hello World tutorial
 
-## Events emitted
+For a simple, gentler, introduction to Lagom, have a look at the [Hello World Tutorial](http://www.lagomframework.com/documentation/1.3.x/scala/GettingStarted.html) in the docs. 
```

Comment:
Hello World is not a tutorial but a Gitter8 seed. This statement could confuse users making them think there's more when they click on the link than what there really is.